src/extension.ts

```diff
@@ -48,12 +48,21 @@ function isError(response: any): response is ErrorShape {
   );
 }
 
-export function activate(context: ExtensionContext) {
+export interface VSCodeGraphQLExtension {
+  outputChannel: OutputChannel;
+  client: LanguageClient;
+  LanguageServerCommands: typeof LSCommands;
+  LanguageServerNotifications: typeof LSNotifications;
+  LanguageServerRequests: typeof LSRequests;
+}
```

Comment:
If an extension's `activate` function returns a value, it can be used by other extensions - or in this case, tests.

Seen in https://github.com/microsoft/vscode/blob/d90ab31527203cdb15056df0dc84ab9ddcbbde40/extensions/git/src/test/smoke.test.ts#L55